parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -340,6 +344,18 @@ ensureTeleportd() {
     systemctlEnableAndStart teleportd || exit $ERR_SYSTEMCTL_START_FAIL
 }
 
+ensureArtifactStreaming() {
+  systemctl start acr-mirror.service
+  sudo /opt/acr/tools/overlaybd/install.sh
```

Comment:
is it possible these could hang?

Reply:
unlikely but possible

Reply:
added an error code for this  - overall error for ensureArtifactStreaming was present after the parent function call too